x/oracle/keeper/grpc_query.go

```diff
@@ -129,3 +129,21 @@ func (q queryServer) GetPrices(goCtx context.Context, req *types.GetPricesReques
 		Prices: prices,
 	}, nil
 }
+
+// RemovedCPs returns the number of currency pairs removed in the previous block.
```

Comment:
Is there a reason we need to expose this as a gRPC query? This feels un-necesssary given that our dependencies aren't on the query server but rather the keeper itself.